src/TimeTracker.Library/Services/TimeEntryService.cs

```diff
@@ -20,15 +20,13 @@ public TimeEntryService(in Guid userId, TimeTrackerDbContext db)
             this.db = db;
         }
 
-        public async Task<Guid> CreateBillableTimeEntry(DateTime date, double hours, int billableClientId,
-            int projectId)
+        public async Task<Guid> CreateBillableTimeEntry(DateTime date, double hours, int projectId)
         {
             VerifyHoursBeforeAdding(date, hours);
 
             var model = new TimeEntry
             {
                 TimeEntryId = Guid.NewGuid(),
-                BillingClientId = billableClientId,
```

Comment:
any specific reason for taking out the billing client id?

Reply:
didn't make sense b/c already have access to the billingClient via the ProjectId, ie. projects belong to billing clients. Gonna remove from Model as well and do the db migration